package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "dev": "yarn docs",
     "build": "vite build",
+    "watch": "yarn build --watch",
```

Comment:
Why it's needed?

Reply:
I wanted changes in `@swimm/ui` to be reflected when working in the webapp. To make it work I had to:
1. In the webapp repo, run `yarn add link:$PATH_TO_SWIMM_UI` and then `yarn install`.
2. Add a `watch` script so code changes will trigger a build.

Did I miss an existing way to do this?